### DISCUSSION TITLE
Browser addon UI difference

### DISCUSSION THREAD

**Discussion by mdsupport** [dani-garcia/vaultwarden#5334] [Q&A] [answered]
Browser UI (edge, firefox) seems to change for accounts in vaultwaden vs bitwarden.com. 

Am I missing any setting or this is expected?

![image](https://github.com/user-attachments/assets/cbc9ff8d-2bf9-44c6-b082-f95af32240c9)

![image](https://github.com/user-attachments/assets/af4c0e50-bf2c-4d4c-816c-ef59ac4956f7)

**Comment by rushout912** [comment, marked as answer]
You need to enable the `extension-refresh` experimental feature flag from environment variables template file.
